### DISCUSSION TITLE
该网站暂无法访问 cn-sy-dx-1.natfrp.cloud

### DISCUSSION THREAD

**Discussion by czhhbp** [natfrp/wiki#183] [使用问题]
我用腾讯云的域名解析了http隧道开通后返回的链接： cn-sy-dx-1.natfrp.cloud，但是访问本人域名或者直接访问cn-sy-dx-1.natfrp.cloud时都提示下图：
![image](https://user-images.githubusercontent.com/32388558/195394219-8e75ca91-c4dc-41db-b437-bbebbceadfa1.png)
![image](https://user-images.githubusercontent.com/32388558/195396308-bd1222d5-0b6d-4ef9-8b22-2aacf1ee86cd.png)

如果建立tcp连接，自动https能正常访问内网端口。
是不是并不是我这边的问题，只是frpc那边提供的域名有问题。
域名是摸索着刚申请通过，ssl证书已签发未部署。
不知是什么原因导致的这个问题。

**Comment by musnows** [comment]
你这个是国内的穿透节点

如果你域名没有备案（不是实名认证，是去`ipc备案`，腾讯云右上角有这个服务的）就会不允许你映射域名的，这个是国内的规定
* 解决办法：使用免费jp的节点穿透试一试

如果你的域名已经完成了ipc备案，就在这里👇点一下验证备案，才能正常映射域名到国内的穿透节点
![image](https://user-images.githubusercontent.com/87506951/196016527-6001ca38-874a-4be2-b567-2c80b6cacdf7.png)
如果你备案了也设置了映射，请忽略我这个回答，我也不知道啥情况

**Reply by czhhbp** [reply]
![image](https://user-images.githubusercontent.com/32388558/198862429-1c0ae977-e401-4e36-8dbf-31a1d1bca675.png)
个人备案需要有云服务器，可是如果有云服务器，也就不需要穿透服务了，两者矛盾了。我去试试jp节点吧

**Reply by Lurito** [reply]
可以咸鱼上花10块钱左右买一个备案资源码，阿里云的只要2块钱
